src/components/footer/footer.css

```
@@ -0,0 +1,21 @@
+footer {
+  background-color: #031f3d;
+  width: 100vw;

```

porcentagem ja nao resolve aqui?

```suggestion
  width: 100%;

```